quora-service/src/main/java/com/upgrad/quora/service/business/QuestionService.java

```diff
@@ -1,10 +1,15 @@
 package com.upgrad.quora.service.business;
 
+import com.upgrad.quora.service.common.ErrorMessage;
 import com.upgrad.quora.service.dao.QuestionDao;
+import com.upgrad.quora.service.entity.AnswerEntity;
```

Comment:
remove this import, if we are not using AnswerEntity anywhere in QuestionService

Reply:
Made changes as per code review